src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -486,6 +486,13 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                     getString(R.string.app_name)
             ));
             pAboutApp.setSummary(String.format(getString(R.string.about_version), appVersion));
+            pAboutApp.setOnPreferenceClickListener(preference -> {
+                String commitUrl = BuildConfig.GIT_REMOTE + "/commit/" + BuildConfig.COMMIT_SHA1;
```

Comment:
I like this, the link is useful 👍

Reply:
Exactly, even when you forked repo and build your current personal fork build, it points to your fork.
Build an official build points to upstream owncloud/android repo